src/tests/Regressions/coreclr/GitHub_49982/test49982.csproj

```diff
@@ -8,6 +8,7 @@
 
     <!-- This is an explicit crossgen test -->
     <AlwaysUseCrossGen2>true</AlwaysUseCrossGen2>
+    <NativeAotIncompatible>true</NativeAotIncompatible>
```

Comment:
Why is this test failing? It does not seem to be doing anything particularly problematic.

Reply:
The test is marked as crossgen specific on the line above and will not run in any other combination (in fact, AlwaysUseCrossgen2 will break any other test configuration).

Reply:
Would it be better to set `NativeAotIncompatible` to true in the top level `Directory.Build.targets` when AlwaysUseCrossGen2 is set so that we do not have to do that for every tests?

Reply:
Yes, looks better.